v1/metrics.yaml

```diff
@@ -81,12 +81,12 @@ paths:
           required: true
         - name: start
           in: path
-          description: The date of the first day to include, in YYYYMMDD format
+          description: The date of the first day to include, in YYYYMMDD or YYMMDDHH format
```

Comment:
Not sure where YYMMDDHH is coming from. I think it should be YYYYMMDDHH instead. Otherwise how would the API know if you are using YYYYMMDD or YYMMDDHH? It's the same amount of numbers. In other words, YYYYMMDD[HH]. The HH part is optional. The response seems to always include it,
 though, BTW.

Reply:
Yeah, you are right. It was late yesterday, when I made the commit :-)